### DISCUSSION TITLE
Is there a way to close preview window by clicking mouse elsewhere?

### DISCUSSION THREAD

**Discussion by SamPosh** [rmagatti/goto-preview#99] [Q&A] [answered]
In Mason ,If we click the mouse in an area which is not float window then the float window will be automatically close. But with the preview window it is available ,is there a plan or an alternate way to achieve it?
(Currently i am assinging keymap gP to close the float window)

**Comment by rmagatti** [comment, marked as answer]
If you use "peak" windows. i.e
```lua
focus_on_open = false;
dismiss_on_move = true; -
```

When you click somewhere outside of the window, it should trigger a CursorMoved event that closes the preview window.
